src/SourceLocator/SourceStubber/PhpStormStubs/CachingVisitor.php

```diff
@@ -135,23 +143,35 @@ public function enterNode(Node $node): ?int
     }
 
     /**
-     * @return array<string, Node\Stmt\ClassLike>
+     * {@inheritDoc}
+     */
+    public function leaveNode(Node $node)
```

Comment:
Should/can we declare `:void` here? :thinking:

Reply:
https://github.com/nikic/PHP-Parser/blob/master/lib/PhpParser/NodeVisitor.php#L39-L58

So I think this is right.